src/Service/Mailer/MailerProvider.php

```diff
@@ -111,6 +111,25 @@ public function sendSignalementValidationWithAutotraitement(Signalement $signale
         $this->send($message);
     }
 
+    public function sendSignalementValidationWithEntreprisesPubliques(Signalement $signalement): void
+    {
+        $emailOccupant = $signalement->getEmailOccupant();
+        $link = $this->urlGenerator->generate('app_suivi_usager_view', ['uuidPublic' => $signalement->getUuidPublic()]);
+        $linkEntreprisesPubliques = $this->parameterBag->get('base_url');
```

Comment:
Sans faire ça, j'ai une url en /entreprises-labellisees sans le nom de domaine (alors que c'est ok pour le lien juste au-dessus). Preneur d'idées :)

Reply:
tu as essayé d'utiliser ABSOLUTE_URL ? (par défaut c'est ABSOLUTE_PATH, donc ça pourrait changer un truc)

Reply:
Super curieux... j'ai essayé, et ça marche  mais c'est étrange : 

![image](https://github.com/MTES-MCT/stop-punaises/assets/33071753/090bcd24-6d92-48a6-906e-955bdafd7963)

Dans le profiler, c'est ok pour le lienEntreprises, mais pas pour $linl : 

![image](https://github.com/MTES-MCT/stop-punaises/assets/33071753/7132b40e-93d2-4666-a043-0fab980a54b7)

Et dans le mail ça ne change rien par rapport à ce que tu as fait : 

![image](https://github.com/MTES-MCT/stop-punaises/assets/33071753/788940f2-ba22-4af7-ab8f-7b9b2e13294f)

Reply:
TROUVE ! 

c'est après qu'on retravaille $link, dans MessageFactory : 

`
        if (isset($parameters['link'])) {
            $parameters['link'] = $this->baseUrl.$parameters['link'];
        }
`

Reply:
Oui j'avais fait ça pour également générer des urls absolues les mails envoyé à travers des commandes. (du coup c'est généralisé)

[D'ailleurs j'ai su après que c'était possible avec un petite configuration](https://symfony.com/doc/current/routing.html#generating-urls-in-commands)

Reply:
aaaaah ok !
bon pour l'instant, je vais mettre un "OR" dans le isset détecté par Hélène... Merci bien !